extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -676,18 +714,67 @@ class QueueProcessor extends EventEmitter {
             }
         }
         if (task) {
-            this.logger.debug('source entry is being pushed',
+            this.logger.debug('replication entry is being pushed',
               { entry: sourceEntry.getLogInfo() });
             return this.taskScheduler.push({ task, entry: sourceEntry,
                                              kafkaEntry },
                                            sourceEntry.getCanonicalKey(),
                                            done);
         }
-        this.logger.debug('skip source entry',
+        this.logger.debug('skip replication entry',
                           { entry: sourceEntry.getLogInfo() });
         return process.nextTick(done);
     }
 
+    /**
+     * Process an action entry from the data mover topic
+     *
+     * @param {object} kafkaEntry - action entry generated by client services
+     * @param {string} kafkaEntry.key - kafka entry key
+     * @param {string} kafkaEntry.value - kafka entry value
+     * @param {function} done - callback function
+     * @return {undefined}
+     */
+    processDataMoverEntry(kafkaEntry, done) {
+        const actionEntry = ActionQueueEntry.createFromKafkaEntry(kafkaEntry);
+        if (actionEntry.error) {
+            this.logger.error('error processing source entry',
+                              { error: actionEntry.error,
+                                entry: kafkaEntry.value });
+            return process.nextTick(() => done(errors.InternalError));
+        }
+        if (actionEntry.skip) {
```

Comment:
Actually the `error` field above is not the one mentioned in this doc: it's the conventional result from `createFromKafkaEntry()` when there is an error to *parse the kafka entry*. If there's no error, then we get an `ActionQueueEntry` object that may contain an error result retrieved with `getError()`, but not accessible directly with an `actionEntry.error` attribute.

For the "skip" it's used in CRR code to skip canary messages, since I kept the same mechanism (useful for pause/resume) I kept using the same convention here than in the other CRR queue.